### ISSUE TITLE
Mobile App - create a login page within the mobile app

### ISSUE THREAD

**Issue by digitaldrummerj** [HTBox/allReady#1097]
Currently the mobile app uses an in-app-browser to launch the mobile web sites login page.  This does not provide the best user experience as the user is then able to navigate around the mobile web page and is not longer really uses the mobile application.

**Comment by digitaldrummerj** [comment]
I am working on this

**Comment by digitaldrummerj** [comment]
having CORS issues with the code that is currently commited when running the mobile app in a web browser vs on a device.  Need to implement the ionic proxies solution to fix that issue.  [http://blog.ionic.io/handling-cors-issues-in-ionic/](http://blog.ionic.io/handling-cors-issues-in-ionic/)

**Comment by tonysurma** [comment]
still open @digitaldrummerj ?

**Comment by digitaldrummerj** [comment]
Yes still open.  I haven't been able to get back to it.  Right now it only supports email/password and doesn't support the social logins.

**Comment by BillWagner** [comment]
@digitaldrummerj If there's work in progress, can you share it? We're at an event in London, and some folks are looking to help on this.

**Comment by jonparker** [comment]
I think we can assume that any work done on this will need to be redone after #2031 
I am happy to work on this ticket basing it off the work @SteveKennedy has done on #2031.
